ui/pages/signup/redirect/index.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react'
+
+export default function SignupRedirect() {
```

Comment:
If `/api/signup/session` returned a 3xx response code with a location header, could we use that to redirect instead of this extra page in the UI?

Not sure what the trade-offs are there